src/twister2/specification_processor.py

```diff
@@ -92,7 +93,7 @@ def process(  # type: ignore[return]
     ) -> YamlTestSpecification | None:
         test_spec_dict = self.prepare_spec_dict(platform, scenario)
         test_spec = self.create_spec_from_dict(test_spec_dict, platform)
-        if not should_be_skip(test_spec, platform):
+        if not should_be_skip(test_spec, platform, self.twister_config.used_toolchain_version):
```

Comment:
Consider passing `self.twister_config` to `should_be_skip` (not only `used_toolchain_version` variable).
In PR #108 I added another should_skip* methods, that need other variables from twister_config.

Reply:
Changed as you suggested